sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AuthTokenContext.java

```diff
@@ -239,6 +239,15 @@ public String getHomeAccountId() {
         return lastEntry != null ? lastEntry.getHomeAccountId() : null;
     }
 
+    /**
+     * Gets current account ID value.
+     *
+     * @return unique identifier of the account.
+     */
+    public String getAccountId() {
+        return getHomeAccountId() == null ? null : getHomeAccountId().substring(0, 36);
```

Comment:
Need to do it exactly like in this PR: https://github.com/Microsoft/AppCenter-SDK-Android/pull/1047/files#diff-21ae9744d09e2b315728e2a1e4c48ac2R53  (if you keep same modification same place there will be no conflict).